Tasks/Krasikov/task1.py

```diff
@@ -0,0 +1,20 @@
+from decimal import *
```

Comment:
oh, try not to use this kind of import, like from smtng import * until we deal with it on a lesson